tools/build_config/build.ps1

```diff
@@ -237,7 +237,7 @@ if ($print_versions) {
   Write-Versions
 }
 
-if ($configure) {
+if ($configure -or -not (Test-Path -Path "${build_dir}/CMakeCache.txt" -PathType Leaf)) {
```

Comment:
What is this additional test for?

This appears to run configure if it's needed not if the configure flag is set. This is a major shift in the way flags works.

Reply:
This is to keep it in line with the Herbert one, where we test if CMakeCache exists and if not run configure so that users didn't have to specify `-configure -build` when building from scratch.